README.md

```diff
@@ -96,6 +59,27 @@ Following are a list of modifiable paths:
 
 4. After registering request-generator, hover over it and click Launch.
 
+### Workflow
+
+The Request Generator's main purpose is to provide the capability to send CDS hooks to and receive/display cards from a CDS service like the [REMS Admin](https://github.com/mcode/rems-admin). After launching the app, the main workflow consists of selecting a patient, selecting a medication for that patient, and then submitting the medication request to the REMS admin.  The REMS admin will respond with a set of cards indicating whether the selected medication has a REMS program, with links to fill out necessary forms if it does have one. The workflow then continues from the request generator to the [REMS SMART on FHIR app](https://github.com/mcode/rems-smart-on-fhir), which handles filling out forms and fulfilling requirements.
+
+The Request Generator also manages tasks, which can be used to defer forms to be completed later, or to assign forms to specific parties. Tasks can be handled in the `tasks` tab and can be created from cards that are returned from the REMS Admin. Some cards will have a suggestion to create a task for completing a form, and clicking on the suggestion button will automatically create a task resource.
```

Comment:
Maybe we should add something about sending the medication PIMS, checking pharmacy status, and ETASU status?